packages/framer-motion/src/projection/node/HTMLProjectionNode.ts

```diff
@@ -23,4 +23,6 @@ export const HTMLProjectionNode = createProjectionNode<HTMLElement>({
     resetTransform: (instance, value) => {
         instance.style.transform = value ?? "none"
     },
+    checkIsScrollRoot: (instance) =>
+        Boolean(window.getComputedStyle(instance).position === "fixed"),
```

Comment:
```suggestion
{
        const position = window.getComputedStyle(instance).position
        return Boolean(position  === "fixed" || position  === "sticky")
}
```

Reply:
By adding this change, it will also fix `sticky` positioned elements that use the `body` scroll.
As forthe other sticky elements, they will be broken either way like before so its worth having this change here

Reply:
ah true, the scroll position will not help here.
I usually check if the sticky is stickying using intersectionobserver but that will kill the performance of a generic implementation